ansible/openstack-ratings.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Generate OpenStack software ratings
+  hosts: localhost
+  tags:
+    - ratings
+  roles:
+    - role: stackhpc.openstack.os_ratings
+      os_ratings_hashmap_field_mappings: "{{ openstack_ratings_hashmap_field_mappings }}"
+      os_ratings_hashmap_service_mappings: "{{ openstack_ratings_hashmap_service_mappings }}"
+      os_ratings_venv: "{{ openstack_venv }}"
+      os_ratings_upper_constraints_file: "{{ openstack_ratings_upper_constraints_file }}"
+      os_ratings_environment: "{{ openstack_ratings_environment }}"
```

Comment:
I don't think these two are necessary. The default upper constraints file should be fine. It's possible we'd need to set the environment in specific cases, but I think we can leave it unset for now. Or if we do want to set it then it should be to a shared `openstack_environment` variable with a default of `{}` in `ansible/group_vars/all/openstack-auth`.

Reply:
Removed them.